pkg/controllers/multicluster/suite_test.go

```diff
@@ -37,8 +37,14 @@ var _ = BeforeSuite(func() {
 	logf.SetLogger(zap.New(zap.WriteTo(GinkgoWriter), zap.UseDevMode(true)))
 
 	By("bootstrapping test environment")
+
+	crds := []string{
+		filepath.Join("..", "..", "..", "config", "crd", "bases", "multicluster.x-k8s.io_serviceexports.yaml"),
+		filepath.Join("..", "..", "..", "config", "crd", "bases", "multicluster.x-k8s.io_serviceimports.yaml"),
+	}
+
```

Comment:
should we have suite_test.go to verify clusterproperty CRDs?

Reply:
As per the original comment up top: 
_In suite_test.go, changes needed to be made to not include the clusterproperty CRD for testing. Special tests will need to be constructed to manually add the annotation to be allowed to use '.k8s.io' suffix._

Reply:
Can you elaborate further? What changes are needed.

Reply:
[See issue. ](https://github.com/aws/aws-cloud-map-mcs-controller-for-k8s/issues/172)